pkg/kubelet/apis/config/v1beta1/defaults.go

```diff
@@ -36,7 +36,7 @@ const (
 	DefaultIPTablesMasqueradeBit = 14
 	DefaultIPTablesDropBit       = 15
 	DefaultVolumePluginDir       = "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/"
-
+	DefaultPodLogsDir            = "/var/log/pods"
```

Comment:
how will this default work on Windows?

I don't think we need this:
https://github.com/kubernetes/kubernetes/blob/60d73ba75148483d150622a4d8b6e122e189ceab/cmd/kubeadm/app/componentconfigs/kubelet_windows.go#L58 but maybe something similar. Not 100% sure

cc: @marosset

Reply:
Also I'm not sure if substitutes like `%SOME_VAR%` are supported on windows for other directory names.

Reply:
@claudiubelu can you comment on this?
Windows APIs allow for `%SOME_VAR%` in paths but I'm not sure about golang (would need to test it out)

We do have a PROW job that runs unit tests on Windows nodes.
https://testgrid.k8s.io/sig-windows-presubmit#pull-windows-unit-master
We can also use this for validation :)

Reply:
@claudiubelu  updates on this? It would be nice to have some best practices how to handle defaults for paths between linux and windows

Reply:
I did test and `os.MkdirAll('/var/log/pods')` creates "c:\\var\\log\pods" on Windows. It seems a bit magical and I would have expected a similar translation, e.g.

```
mutateStringField("podLogsDir", &cfg.PodLogsDir)
```
Any reason not to do this?

Reply:
Seems like it works as is :)
(But I don't have a Win environment to do more careful testing of this).